swarm_copy/app/dependencies.py

```diff
@@ -230,7 +232,7 @@ async def get_vlab_and_project(
 
 
 def get_starting_agent(
-    _: Annotated[None, Depends(get_vlab_and_project)],
+    # _: Annotated[None, Depends(get_vlab_and_project)],
```

Comment:
I just did this since the token generating service was down at the time of writing this PR.

We should not merge that